account_financial_report/readme/DESCRIPTION.rst

```diff
@@ -18,3 +18,6 @@ currency balances are not available.
 Invoicing / Settings / Invoicing / OCA Aged Report Configuration you will be able to set
 dynamic intervals that will appear on the Aged Partner Balance.
 For further information, check CONFIGURE.rst
+
+Add new grouped by field to export Open items report grouped by partner salesperson.
```

Comment:
This is not something suitable for the general description. This should go on USAGE.rst, but as there's none, better to not put anything.